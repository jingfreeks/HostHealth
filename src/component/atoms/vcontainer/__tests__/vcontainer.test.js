import React from 'react';
import {render} from '@testing-library/react-native';
import Vcontainer from '../vcontainer';

describe('VContainer Component', () => {
  it('Should work as expected to get snapshot', () => {
    const all = render(<Vcontainer />);
    expect(all.toJSON()).toMatchSnapshot();
  });
});
