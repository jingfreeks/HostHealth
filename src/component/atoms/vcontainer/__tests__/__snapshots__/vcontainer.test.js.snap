// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VContainer Component Should work as expected to get snapshot 1`] = `
<RCTSafeAreaView
  style={
    [
      {},
    ]
  }
>
  <View
    alignType="flex-start"
    style={
      [
        {
          "alignItems": "flex-start",
          "alignSelf": "flex-start",
          "flexDirection": "column",
        },
      ]
    }
  />
</RCTSafeAreaView>
`;
